### PR TITLE
use tmp for make_video outputs. test notebook.

### PR DESCRIPTION
**Pull request by orm011**
This change is [Reviewable](https://reviewable.io/reviews/mkornacker/pixeltable/68)

### CODE REVIEW

File: pixeltable/functions/__init__.py [hunk 1]
```diff
@@ -131,7 +131,8 @@ def update(self, frame: PIL.Image.Image) -> None:
         if frame is None:
             return
         if self.container is None:
-            self.out_file = Path(os.getcwd()) / f'{Path(tempfile.mktemp()).name}.mp4'
+            (_fd, output_filename) = tempfile.mkstemp(suffix='.mp4', dir=str(env.Env.get().tmp_dir))
```
Comment (mkornacker) [inline pixeltable/functions/__init__.py]: (_, ...) since it's unused?

File: docs/tutorials/object_detection_in_videos.ipynb [hunk 2]
```diff
@@ -329,7 +329,76 @@
    "id": "cae0de07",
    "metadata": {},
    "source": [
-    "We loaded a video that shows a busy intersection in Bangkok. Let's look at the first frame:"
+    "We loaded a video that shows a busy intersection in Bangkok."
+   ]
+  },
+  {
```
Comment (mkornacker) [inline docs/tutorials/object_detection_in_videos.ipynb:335]: What's this for?

**Review by orm011 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mkornacker/pixeltable/68)* status: 0 of 2 files reviewed, 2 unresolved discussions (waiting on @mkornacker)

*[a discussion](https://reviewable.io/reviews/mkornacker/pixeltable/68#gh-1460134644:-NoZg-Zn8kZ4cjCXIGxC:b-896fix) (no related file):*
<details><summary><i>Previously, mkornacker (Marcel Kornacker) wrote…</i></summary><blockquote>

*(Reviewable was [unable](https://github.com/Reviewable/Reviewable/issues/14) to map [this GitHub inline comment thread](https://github.com/mkornacker/pixeltable/pull/68#discussion_r1460134644) to the right spot — sorry!)*

(_, ...) since it's unused?
</blockquote></details>

Done.

___
*[a discussion](https://reviewable.io/reviews/mkornacker/pixeltable/68#gh-1460134761:-NoZg0-3DzJ5EjGkWP0L:b-yw73e7) (no related file):*
<details><summary><i>Previously, mkornacker (Marcel Kornacker) wrote…</i></summary><blockquote>

*(Reviewable was [unable](https://github.com/Reviewable/Reviewable/issues/14) to map [this GitHub inline comment thread](https://github.com/mkornacker/pixeltable/pull/68#discussion_r1460134761) to the right spot — sorry!)*

What's this for?
</blockquote></details>

i split the sentence into two parts, one in a cell where we show the video table (bc now the video is shown properly), and another where we check the frame (same as before)

___

**Comment by mkornacker** [comment]
Let's close this, I already incorporated the fix into
commit 6f74f257e5b2ddea815c3be3e2333765234e6d4f
Author: Marcel Kornacker <marcelk@gmail.com>
Date:   Wed Dec 20 14:35:54 2023 -0800

    Reimplementation of snapshots
    
    Snapshots create a DAG of table versions, which needs to be captured by
    introducing a level of indirection over TableVersions (which is the purpose
    of the new class TableVersionPath). The previous implementation relied on
    TableVersion.base, which is incorrect, because it records a tree, not the required DAG.
    
    Also:
    - moved catalog loading into new class Catalog
    - removed TableVersion parameter from Expr._from_dict(): the global catalog
      is now used for looking up catalog structures
    - updated the docs

**Pull request closed without merging**